<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CColorsDialog</class>
 <widget class="QDialog" name="CColorsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>261</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Colors and styling</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0,0">
     <property name="spacing">
      <number>4</number>
     </property>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Theme</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="_cbTheme"/>
     </item>
     <item>
      <widget class="QPushButton" name="_btnCopy">
       <property name="text">
        <string>New copy...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="_btnDelete">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Window background</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QToolButton" name="btnWindowBackgroundColor">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Text background</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QToolButton" name="btnTextBackgroundColor">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Text</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QToolButton" name="btnTextColor">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Pivot</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QToolButton" name="btnPivotColor">
       <property name="styleSheet">
        <string notr="true">background-color: white;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>87</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CColorsDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CColorsDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>199</x>
     <y>279</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
